views/default/css/elements/misc/spinner.css

```diff
@@ -0,0 +1,34 @@
+/* ***************************************
+	FIXED POSITION AJAX INDICATOR
+*************************************** */
+
+.elgg-spinner {
+    position: fixed;
+    top: 0;
+    left: 50%;
+    z-index: 10000;
+
+    box-sizing: border-box;
+    border: 1px solid #CCC;
+    border-top: 0;
+    border-radius: 0 0 10px 10px;
```

Comment:
Let's increase to 0 0 ```15px 15px```, that way it will follow the radius of the spinner graphic